io/plugins/eu.esdihumboldt.hale.io.csv.ui/src/eu/esdihumboldt/hale/io/csv/ui/InstanceExportConfigurationPage.java

```diff
@@ -122,23 +119,17 @@ public void widgetSelected(SelectionEvent e) {
 				}
 			});
 
-			Label separatorLabel = new Label(page, SWT.NONE);
-			separatorLabel.setText("Warning! Feature types with no data are not selectable");
+			Table table = new Table(chooseFeatureTypes,
+					SWT.CHECK | SWT.MULTI | SWT.V_SCROLL | SWT.H_SCROLL);
 
-			// Set the text colour of the label to yellow
-			Color greyLabel = PlatformUI.getWorkbench().getDisplay()
-					.getSystemColor(SWT.COLOR_DARK_GRAY);
-			separatorLabel.setForeground(greyLabel);
+			featureTypeTable = new CheckboxTableViewer(table);
 
-			table = new Table(chooseFeatureTypes, SWT.CHECK | SWT.MULTI | SWT.SCROLL_PAGE);
-			table.setHeaderVisible(false);
-			table.setLinesVisible(false);
-			table.setBackground(PlatformUI.getWorkbench().getDisplay()
-					.getSystemColor(SWT.COLOR_LIST_BACKGROUND));
-			GridDataFactory groupData = GridDataFactory.fillDefaults().grab(true, false);
-			groupData.applyTo(table);
+			// Configure the layout for the table
+			GridData tableGridData = new GridData(SWT.FILL, SWT.FILL, true, true);
+			tableGridData.heightHint = 300; // Minimum height for the table
+			tableGridData.widthHint = 300; // Minimum width for the table
+			table.setLayoutData(tableGridData);
```

Comment:
Fine for me like that, but I think using `GridDataFactory` would be generally preferable for applying the settings.

Reply:
Thank you for the suggestion. I have updated the code accordingly.